### PR TITLE
Create function sort book

### PR DESCRIPTION
**Pull request by at-luanle**
## Description
Sort for book page by author, name, avg rating, total borrowing
## Reference Requirements
- Product Backlog ID: #PBI-7
- Task ID: #PBI-7.2

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:
app/Http/Controllers/Admin/BookController.php
app/Model/Book.php
resources/views/backend/books/list.blade.php
-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)
screenshot from 2018-01-08 16 31 42

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

**Comment by at-nghiavo** [comment]
LGTM

File: app/Model/Book.php [hunk 1]
```diff
@@ -107,4 +111,19 @@ public function getTotalBorrowAttribute()
     {
         return $this->borrowings->count();
     }
+
```
Comment (at-huydq) [inline app/Model/Book.php]: xoá bớt 1 line trống

File: app/Model/Book.php [hunk 2]
```diff
@@ -107,4 +111,19 @@ public function getTotalBorrowAttribute()
...
+
+    /**
+     * Declare table sort
+     *
+     * @var array $sortable table sort
+     */
+    public $sortable = ['id', 'name', 'author', 'avg_rating'];
```
Comment (at-huydq) [inline app/Model/Book.php]: các biến định nghĩa `$fillable` vs `$sortableAs`  anh move lên trên giúp em nhé. để dưới `$fillable`

**Review by ghost — COMMENTED** [review]
please check my comment

**Comment by at-huydq** [comment]
LGTM.

**Pull request merged**